test/alternator/test_batch.py

```diff
@@ -311,6 +311,21 @@ def test_batch_get_item_projection_expression(test_table):
         expected_items = [{k: item[k] for k in wanted if k in item} for item in items]
         assert multiset(got_items) == multiset(expected_items)
 
+# ExpressionAttributeNames can be used to define aliases for attribute names for ProjectionExpression
+def test_batch_get_item_projection_expression_with_substitute_name(test_table):
```

Comment:
Nitpick: I would stick to DynamoDB terminology and avoid inventing our our own terminology. Here the feature is called "expression attribute names",  so in my opinion using phrases like "aliases" or "substitute names" just confuses things. I think you could have more simply had a comment like
```
# Test GetItem with ProjectionExpression using an attribute from ExpressionAttributeNames
```

Reply:
ok